examples/connext_dds/settings.gradle

```diff
@@ -0,0 +1,41 @@
+rootProject.name = 'rticonnextdds-examples'
+
+includeBuild '../../resources/gradle_plugin'
+include 'asynchronous_publication:java'
```

Comment:
Do we have a guide that shows how to add new examples of Java? This will be helpful for people to know what files to modify when adding a new example.

Reply:
This was added to the CONTRIBUTING.md but I have to update it.